### ISSUE TITLE
wrong parameter order in `create_namespaced_binding_binding` method

### ISSUE THREAD

**Issue by sebgoa** [kubernetes-incubator/client-python#175]
the doc shows that the `name` should be first:

https://github.com/kubernetes-incubator/client-python/blob/master/kubernetes/docs/CoreV1Api.md#create_namespaced_binding_binding

but the call actually takes the `body` as first argument.

**Comment by mbohlool** [comment]
Not sure where the call actually takes `body` first. Looking at the code in this file:

https://github.com/kubernetes-incubator/client-python/blob/84bf6cc9dc206cb3314a5b4d2fa1a9c2fa2c5137/kubernetes/client/apis/core_v1_api.py

(can't link to the line, the file is large. search for create_namespaced_binding_binding)

The function defined like this:

```python
def create_namespaced_binding_binding(self, name, namespace, body, **kwargs):
        """
        create binding of a Binding
        This method makes a synchronous HTTP request by default. To make an
...
```

**Comment by sebgoa** [comment]
I have been looking at the code, and I don't see anything obviously wrong with it.

But this can be reproduced. Create a pod on minikube with the following manifest:

```
apiVersion: v1
kind: Pod
metadata:
  name: foobar-sched
spec:
  schedulerName: foobar
  containers:
  - name: redis
    image: redis
```

Then Try to create a binding:

```
$ ipython
Python 2.7.12 (default, Oct 11 2016, 14:42:58) 
Type "copyright", "credits" or "license" for more information.
...
In [1]: from kubernetes import client, config

In [2]: config.load_kube_config()

In [3]: v1=client.CoreV1Api()

In [4]: body=client.V1Binding()

In [5]: target=client.V1ObjectReference()

In [6]: target.kind="Node"

In [7]: target.apiVersion="v1"

In [8]: target.name="minikube"

In [9]: meta=client.V1ObjectMeta()

In [10]: meta.name="foobar-sched"

In [11]: body.target=target

In [15]: name="foobar-sched"

In [17]: body.metadata=meta

In [20]: namespace="default"

In [22]: type(body)
Out[22]: kubernetes.client.models.v1_binding.V1Binding

In [23]: v1.create_namespaced_binding_binding(name, namespace, body)
---------------------------------------------------------------------------
ApiException                              Traceback (most recent call last)
<ipython-input-23-373b504f3b3f> in <module>()
----> 1 v1.create_namespaced_binding_binding(name, namespace, body)

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/apis/core_v1_api.pyc in create_namespaced_binding_binding(self, body, name, namespace, **kwargs)
   5269             return self.create_namespaced_binding_binding_with_http_info(body, name, namespace, **kwargs)
   5270         else:
-> 5271             (data) = self.create_namespaced_binding_binding_with_http_info(body, name, namespace, **kwargs)
   5272             return data
   5273 

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/apis/core_v1_api.pyc in create_namespaced_binding_binding_with_http_info(self, body, name, namespace, **kwargs)
   5365                                         _preload_content=params.get('_preload_content', True),
   5366                                         _request_timeout=params.get('_request_timeout'),
-> 5367                                         collection_formats=collection_formats)
   5368 
   5369     def create_namespaced_config_map(self, namespace, body, **kwargs):

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/api_client.pyc in call_api(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, callback, _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    327                                    body, post_params, files,
    328                                    response_type, auth_settings, callback,
--> 329                                    _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    330         else:
    331             thread = threading.Thread(target=self.__call_api,

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/api_client.pyc in __call_api(self, resource_path, method, path_params, query_params, header_params, body, post_params, files, response_type, auth_settings, callback, _return_http_data_only, collection_formats, _preload_content, _request_timeout)
    151                                      post_params=post_params, body=body,
    152                                      _preload_content=_preload_content,
--> 153                                      _request_timeout=_request_timeout)
    154 
    155         self.last_response = response_data

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/api_client.pyc in request(self, method, url, query_params, headers, post_params, body, _preload_content, _request_timeout)
    381                                          _preload_content=_preload_content,
    382                                          _request_timeout=_request_timeout,
--> 383                                          body=body)
    384         elif method == "PUT":
    385             return self.rest_client.PUT(url,

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/rest.pyc in POST(self, url, headers, query_params, post_params, body, _preload_content, _request_timeout)
    273                             _preload_content=_preload_content,
    274                             _request_timeout=_request_timeout,
--> 275                             body=body)
    276 
    277     def PUT(self, url, headers=None, query_params=None, post_params=None, body=None, _preload_content=True,

/opt/local/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/kubernetes/client/rest.pyc in request(self, method, url, query_params, headers, body, post_params, _preload_content, _request_timeout)
    229 
    230         if r.status not in range(200, 206):
--> 231             raise ApiException(http_resp=r)
    232 
    233         return r

ApiException: (400)
Reason: Bad Request
HTTP response headers: HTTPHeaderDict({'Date': 'Mon, 17 Apr 2017 09:43:37 GMT', 'Content-Length': '417', 'Content-Type': 'application/json'})
HTTP response body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"the object provided is unrecognized (must be of type Binding): couldn't get version/kind; json parse error: json: cannot unmarshal string into Go value of type struct { APIVersion string \"json:\\\"apiVersion,omitempty\\\"\"; Kind string \"json:\\\"kind,omitempty\\\"\" } (22666f6f6261722d736368656422)","reason":"BadRequest","code":400}



In [24]: v1.create_namespaced_binding_binding(body, name, namespace)
Out[24]: 
{'api_version': 'v1',
 'kind': 'Status',
 'metadata': {'annotations': None,
              'cluster_name': None,
              'creation_timestamp': None,
              'deletion_grace_period_seconds': None,
              'deletion_timestamp': None,
              'finalizers': None,
              'generate_name': None,
              'generation': None,
              'labels': None,
              'name': None,
              'namespace': None,
              'owner_references': None,
              'resource_version': None,
              'self_link': None,
              'uid': None},
 'target': None}

In [25]: 
```

**Comment by sebgoa** [comment]
ok looks like it is a problem with the 1.0.0 release, but is fixed in the 2.0.0 pre-release

**Comment by mbohlool** [comment]
I will release a beta 2.0 soon.
